File: langs/QR.scala
object QR {
  def main(args: Array[String]): Unit = {
    val data = "https://example.com"
    val url = s"https://api.qrserver.com/v1/create-qr-code/?data=$data&size=240x240"
    println(url)
  }
}

File: langs/qr.dart
// Dart
import 'dart:convert';

void main() {
  final data = 'https://example.com';
  final url = 'https://api.qrserver.com/v1/create-qr-code/?data=${Uri.encodeComponent(data)}&size=240x240';
  print(url);
}
